test.js

```diff
@@ -84,3 +84,38 @@ test('obj arrays', function (t) {
     t.end()
   })
 })
+
+
+test('arrays can emit errors', function (t) {
+  var input = ['a', 'b', new Error('ooops'), 'c']
+  var stream = from(input)
+  var output = []
+  stream.on('data', function (letter) {
+    output.push(letter.toString())
```

Comment:
Sure l I could check that 2 elements have been added to the array before the error is introduced...